app/views/trainees/degrees/_uk_degree_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with model: [ @trainee, @degree ], local: true do |f| %>
+  <h2 class="govuk-heading-l">Degree Details</h2>
+  <%= f.govuk_error_summary %>
+  <%= f.govuk_collection_select :degree_subject, degree_subjects, 
+        :name, :name, label: { text: 'Degree subject', size: 's' },
+        hint: { text: 'Search for the closest matching subject' } %>
+
+  <%= f.govuk_collection_select :institution, institutions, :name, 
+        :name, label: { text: "Institution", size: 's' } %>
+
+  <%= f.govuk_text_field :graduation_year, 
+        label: { text: "Graduation Year", size: 's' }, 
+        width: "one-quarter"  %>
+
+  <%= f.govuk_radio_buttons_fieldset(:degree_grade, 
+        legend: { text: "Degree grade", size: 's' }, 
+        classes: "degree-grade" ) do %>
+
+    <% DEGREE_GRADES.map do |name| %>
+      <%= f.govuk_radio_button :degree_grade, name, label: { text: name }, link_errors: true %>
+    <% end %>
```

Comment:
should this not be part of the helper method?

Reply:
need to pick your brain for that one....

Reply:
I'm assuming these will most likely end up being saved in the db so I'm happy for us to merge this as is